app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -282,7 +282,9 @@ public boolean onCreateOptionsMenu(Menu menu) {
             // TODO: used vectors are not compatible with API 19 and below, change them
             menu.findItem(R.id.notifications).setIcon(ContextCompat.getDrawable(this, R.drawable.ic_notification_white_clip_art));
         } else {
-            menu.findItem(R.id.notifications).setIcon(ContextCompat.getDrawable(this, R.drawable.ic_notification_white_clip_art_dot));
+            final View notification=menu.findItem(R.id.notifications).getActionView();
+            txtViewCount=notification.findViewById(R.id.notification_count_badge);
+            txtViewCount.setText(String.valueOf(1));
```

Comment:
is this value hardcoded?

Reply:
Yes presently, Because I didn't had the number of unread notifications